src/main/java/duke/command/Duke.java

```diff
@@ -0,0 +1,176 @@
+package duke.command;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Scanner;
+
+public class Duke {
+
+
+    public static void main(String[] args) throws IOException {
+
+        List listItem = new List();
+        File file = new File("C:\\Users\\muham\\OneDrive\\Desktop\\NUS MATERIAL\\Sem 5\\CS2113\\IP\\ip\\src\\main\\java\\duke.txt");
+
+
+        BufferedReader br = new BufferedReader(new FileReader(file));
+
+        String st;
+        String taskDesc = "";
+        String dateDesc = "";
+        while ((st = br.readLine()) != null) {
+            String[] commandList = st.split("\\|");
+            try {
```

Comment:
Your if's and else-if's are not consistent. Do consider following the coding standard so that it is easier to read